tf2_ros/include/tf2_ros/buffer_interface.h

```diff
@@ -55,11 +55,10 @@ namespace tf2_ros
     return time_msg;
   }
 
-  inline const tf2::TimePoint& fromMsg(const builtin_interfaces::msg::Time & time_msg)
```

Comment:
For some context here, when working on https://github.com/ros2/geometry2/pull/2, this signature was chosen for compatibility with `getTimestamp` https://github.com/ros2/geometry2/blob/a0896de571fbc9738b71456f15788cbbed1ca6cd/tf2/include/tf2/convert.h#L62 which @tfoote told me returns const ref to allow for chaining.

In particular, changing this signature will break this line: https://github.com/ros2/geometry2/pull/7/files#diff-bb6bd2d938a7c0ed4397920b3d645fd3R65
There are no tests for tf2_geometry_msgs at the moment and these template instantiations often won't give errors unless you're actually using the code. adding the tests is the next thing on my list, I just haven't gotten to it yet.

It may be that the std::move call can be moved into https://github.com/ros2/geometry2/pull/7/files#diff-bb6bd2d938a7c0ed4397920b3d645fd3R65 to adapt for this change (I am not 100% familiar with the intricacies of this).

The std::move was added to fix that compiler warning, but I admit that it wasn't fully tested on Windows. It might be that the signature can be left as it was with some tweaks for it to work on windows. It could also be that it is the way `fromMsg` is being used in this PR that is causing the errors.